csharp/src/IceLocatorDiscovery/PluginI.cs

```diff
@@ -11,11 +11,6 @@ public Ice.Plugin
     create(Ice.Communicator communicator, string name, string[] args) => new PluginI(communicator);
 }
 
-public interface Plugin : Ice.Plugin
```

Comment:
We use this (obviously) internal API in C++ and Java, but not in C#, where it's internal unused code.